### PR TITLE
Create thorgathis.json

### PR DESCRIPTION
**Pull request by Thorgathis**

### CODE REVIEW

File: domains/thorgathis.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Thorgathis website",
+  "repo": https://github.com/Thorgathis/thorgathis.github.io",
```
Comment (plu5) [inline domains/thorgathis.json:3]: psst, you’re missing an open quotation mark (") here

Comment (Thorgathis) [reply domains/thorgathis.json]: I fix

**Review by MaskDuck — APPROVED** [review]
ooo... i think you have discord too?

**Comment by Thorgathis** [comment]
> 

What?

**Pull request merged**